docker-compose-infura.yml

```diff
@@ -0,0 +1,34 @@
+version: "2.1"
+services:
+  plasma-deployer:
```

Comment:
now that we don't use plasma-deployer in standard `docker-compose` this won't get overwritten properly, I think we need to bring the full plasma-deployer description here, referencing the tag that supports infura

Reply:
@InoMurko It seems it should be possible to follow the latest `docker-compose.yml` way of deploying the contracts for infura too. Any red flags if I follow that route? It'll simplify things a lot not having to deploy contracts in different ways.

Reply:
nop, not really! I would love to see us deprecating plasma-deployer completely!